src/scene/morph-instance.js

```diff
@@ -238,18 +239,18 @@ class MorphInstance {
             }
 
             // render quad with shader for required number of textures
-            var shader = this._getShader(usedCount);
+            const shader = this._getShader(usedCount);
             drawQuadWithShader(device, renderTarget, shader, undefined, undefined, blending);
 
         }.bind(this);
 
         // set up parameters for active blend targets
-        var usedCount = 0;
-        var blending = false;
-        var count = this._activeTargets.length;
-        for (var i = 0; i < count; i++) {
-            var activeTarget = this._activeTargets[i];
-            var tex = activeTarget.target[srcTextureName];
+        let usedCount = 0;
+        let blending = false;
+        const count = this._activeTargets.length;
+        for (let i = 0; i < count; i++) {
+            const activeTarget = this._activeTargets[i];
```

Comment:
Could this be:

```
for (const activeTarget of this._activeTargets) {
```

Reply:
Could be. I still use `count` later, so that is needed. but in general old school for is faster, so I tend to keep it.

Reply:
OK, fair enough. I tend to use `for ... in` for init/occasional stuff....and vanilla `for` for per-frame stuff.